lib/jekyll/watcher.rb

```diff
@@ -4,10 +4,10 @@ module Jekyll
   module Watcher
     extend self
 
-    def watch(options)
+    def watch(options, site = nil)
```

Comment:
Wonderful! Would you mind adding a comment that the `site` is only populated in Jekyll v3.2 (upcoming) and up? Thanks!